src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -829,21 +819,11 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with the <see cref="LinearSvmTrainer"/> trainer.
+        /// Create a <see cref="LinearSvmTrainer"/>, which predicts a target using a linear binary classification model
```

Comment:
>Create a <see cref="LinearSvmTrainer"/> [](start = 12, length = 39)

the second overload should say: 
Create a <see cref="LinearSvmTrainer"/> **with advanced options**,